docs/modeling/processes/variables.md

```diff
@@ -27,9 +31,9 @@ The following are the data types that process variables can be set as:
 | File | A [file](../modeling/files.md) uploaded into a process definition or as part of a process instance or task | 
 | JSON | A JSON object | `{"flavor" : "caramel"}` | 
 
-Process variables are stored in the `properties` of the [`<process-name>-extensions.json`](../modeling/projects.md#files) file with unique IDs and can also be viewed through the UI in the **Extensions Editor**: 
+Process variables are stored in the `properties` of the [`<process--definition-name>-extensions.json`](../modeling/projects.md#files) file with unique IDs and can also be viewed through the UI in the **Extensions Editor**: 
```

Comment:
In the linked page (../modeling/projects.md#files) the file still is named `<process-name>-extensions.json`

Reply:
Thanks for the feedback @pmartinezga ! 
This has been updated in https://github.com/Alfresco/docs-activiti-enterprise/pull/55/